docs/data/material/getting-started/templates/sign-in-side/Content.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+
+import AutoFixHighRoundedIcon from '@mui/icons-material/AutoFixHighRounded';
+import ConstructionRoundedIcon from '@mui/icons-material/ConstructionRounded';
+import SettingsSuggestRoundedIcon from '@mui/icons-material/SettingsSuggestRounded';
+import ThumbUpAltRoundedIcon from '@mui/icons-material/ThumbUpAltRounded';
+
+import { SitemarkIcon } from './CustomIcons';
+
+const items = [
+  {
+    icon: <SettingsSuggestRoundedIcon />,
+    title: 'Adaptable performance',
+    description:
+      'Our product effortlessly adjusts to your needs, boosting efficiency and simplifying your tasks.',
+  },
+  {
+    icon: <ConstructionRoundedIcon />,
+    title: 'Built to last',
+    description:
+      'Experience unmatched durability that goes above and beyond with lasting investment.',
+  },
+  {
+    icon: <ThumbUpAltRoundedIcon />,
+    title: 'Great user experience',
+    description:
+      'Integrate our product into your routine with an intuitive and easy-to-use interface.',
+  },
+  {
+    icon: <AutoFixHighRoundedIcon />,
+    title: 'Innovative functionality',
+    description:
+      'Stay ahead with features that set new standards, addressing your evolving needs better than the rest.',
+  },
+];
+
+export default function Content() {
+  return (
+    <Stack
+      flexDirection="column"
+      alignSelf="center"
+      spacing={4}
+      sx={{
+        width: { xs: '100%', sm: '450px' },
```

Comment:
```suggestion
        maxWidth: 450,
```

Let's do this instead